Formula/mongosh.rb

```diff
@@ -27,5 +27,6 @@ def install
   test do
     assert_match "ECONNREFUSED 0.0.0.0:1", shell_output("#{bin}/mongosh \"mongodb://0.0.0.0:1\" 2>&1", 1)
     assert_match "#ok#", shell_output("#{bin}/mongosh --nodb --eval \"print('#ok#')\"")
+    assert_match "all tests passed", shell_output("#{bin}/mongosh --smokeTests 2>&1")
```

Comment:
If we only really care about the exit code you can just do
```ruby
system bin/"mongosh", "--smokeTests"
```
Do the smoke tests fail with Node 19?

Reply:
Quoting myself from #123143:

> We do have `mongosh --smokeTests` and should probably add that to the test script here, but without a full MongoDB server setup to test against, they are pretty limited.

> It probably wouldn’t have caught the issues mentioned above though, since even the full set of smoke tests only tests one specific networking setup.

Unfortunately, no, they don’t.

Reply:
Also, yeah, just checking the exit code would also work here, but that text should also always be present. I don’t have a strong preference about this :)